node/service/src/chain_spec/frequency_rococo.rs

```diff
@@ -158,7 +159,7 @@ pub fn frequency_rococo_testnet() -> ChainSpec {
 		},
 		// Bootnodes
 		vec![
-			"/dns/0.boot.rococo.frequency.xyz/tcp/30333/p2p/5CkhzgoWc21tKSYATXhba2Rj1Nhu6subu8FEfbxPk9GFjGqd".parse().unwrap(),
+			"/dns4/0.boot.rococo.frequency.xyz/tcp/30333/ws/p2p/12D3KooWArmKDbY8Y6XXHGodosWAjRWWxSw5YxWEjSZTBNjJXVSC".parse().unwrap(),
```

Comment:
where do we get this info, and why is it changed?

Reply:
@shannonwells So this is from our new bootnode we setup. The public key on the end `12D3K...` is from a node key that we generated and loaded in.

Some docs about it were added here: https://github.com/LibertyDSNP/frequency/wiki/Adding-Collators#key-structure

Node keys are usually auto-generated, but to put them in things like this you need a stable one.

Also while the bootnodes are part of the spec json, they can be updated without effecting the genesis state.

Reply:
More docs will be added to the docs.frequency.xyz site as well.